build/buildall

```diff
@@ -160,8 +160,19 @@ shift
 
 done
 
+arch=$(uname -m)
+case ${arch} in
+  x86_64|amd64)
+    cpu_arch='amd64';;
+  aarch64|arm64)
+    cpu_arch='arm64';;
+  *)
+    echo "Non-support cpu architecture: ${arch}"; exit 1;;
```

Comment:
Nit, applies to all the places where it was copied.  Not must-fix.
```suggestion
    echo "Unsupported CPU architecture: ${arch}"; exit 1;;
```

Reply:
Fixed